tox.ini

```diff
@@ -28,6 +28,9 @@ commands = pre-commit run --all-files
 [testenv:lint]
 commands =
     pflake8
+    pylint --recursive=y .
+    pylint {toxinidir}/tests/ --disable=E1101,R0913,R0914,W0212,W0613,R0801,F0010
```

Comment:
Is this line really needed? Because previous line is checking also `{toxinidir}/tests/`, if I'm not wrong.

Reply:
You are right. Since I'm excluding tests/ in `pyproject.toml`, this line is not needed.